app/build.gradle

```diff
@@ -21,6 +21,15 @@ android {
                 }
             }
         }
+        debug {
```

Comment:
@depapp I set APK file name is `app-debug.apk` for GitHub Action

Reply:
great @ttpho , thanks 🍻